### PR TITLE
Added book "Getting Started with Java on the Raspberry Pi"

### PR DESCRIPTION
**Pull request by FDelporte**
# Please make sure all below checkboxes have been checked before submitting your PR.

- [x] I have read and understood the [contribution guidelines](https://github.com/thibmaek/awesome-raspberrypi/blob/master/CONTRIBUTING.md).
- [x] This pull request has a descriptive title. *(For example: `Add Raspbian`)*
- The topic I added
  - [x] includes a valid (https) link,
  - [x] includes a concise and on-topic description,
  - [x] mentions if there is only support some devices,
  - [x] is not a duplicate,
  - [x] has been in the correct alphabetical order for its section,
  - [x] is in the form of **Named Link - Description, seperated by commas.**
  - [x] ends with a dot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,6 +218,10 @@ The complete collection of (consumer) Raspberry Pi models consist of:
 - [Raspberry Pi Beginners](https://medium.com/@anshul.ahu/how-to-setup-raspberry-pi-for-beginners-aeedc2cb994a) - Guide to setup the Raspberry Pi for beginners.
 - [The Ultimate Raspberry Pi Security Guide](http://www.nhatqbui.com/assets/TheUltimateRaspberryPiSecurityGuide.pdf) - Extensive guide on security best practices for the Pi ie. exposed to the internet.
 
+### Books
+
+- [Getting Started with Java on the Raspberry Pi (published 2020)](https://webtechie.be/books/) - A lot of info and history about Java itself and how to install and use it on the Raspberry Pi with a lot of example projects.
```
Comment (thibmaek) [inline README.md:223]: Omit the `(published 2020)` part please

Comment (thibmaek) [reply README.md:223]:
```suggestion
- [Getting Started with Java on the Raspberry Pi](https://leanpub.com/gettingstartedwithjavaontheraspberrypi) - Book on how to install & use Java with a lot of example projects.
```

Comment (thibmaek) [reply README.md:223]: I would link to Leanpub instead of a generic /books page

**Review by thibmaek — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (FDelporte) [reply README.md]: thanks for your feedback! both remarks done

**Review by thibmaek — APPROVED** [review]
🤝 Thanks for contributing this resource!

**Comment by FDelporte** [comment]
@thibmaek thanks for sharing!!!

**Pull request merged**